moto/glue/models.py

```diff
@@ -757,6 +771,79 @@ def update_schema(
 
         return schema.as_dict()
 
+    def create_trigger(
+        self,
+        name: str,
+        workflow_name: str,
+        trigger_type: str,
+        schedule: str,
+        predicate: Dict[str, Any],
+        actions: List[Dict[str, Any]],
+        description: str,
+        start_on_creation: bool,
+        tags: Dict[str, str],
+        event_batching_condition: Dict[str, Any],
+    ) -> None:
+        self.triggers[name] = FakeTrigger(
+            name=name,
+            workflow_name=workflow_name,
+            trigger_type=trigger_type,
+            schedule=schedule,
+            predicate=predicate,
+            actions=actions,
+            description=description,
+            start_on_creation=start_on_creation,
+            tags=tags,
+            event_batching_condition=event_batching_condition,
+            backend=self,
+        )
+
+    def get_trigger(self, name: str) -> "FakeTrigger":
+        try:
+            return self.triggers[name]
+        except KeyError:
+            raise TriggerNotFoundException(name)
+
+    def start_trigger(self, name: str) -> None:
+        trigger = self.get_trigger(name)
+        trigger.start_trigger()
+
+    def stop_trigger(self, name: str) -> None:
+        trigger = self.get_trigger(name)
+        trigger.stop_trigger()
+
+    @paginate(pagination_model=PAGINATION_MODEL)
+    def get_triggers(self, dependent_job_name: str) -> List["FakeTrigger"]:  # type: ignore
+        if dependent_job_name:
+            triggers = []
+            for trigger in self.triggers.values():
+                for action in trigger.actions:
+                    if ("JobName" in action) and (
+                        action["JobName"] == dependent_job_name
+                    ):
+                        triggers.append(trigger)
+            return triggers
+
+        return list(self.triggers.values())
+
+    @paginate(pagination_model=PAGINATION_MODEL)
+    def list_triggers(self, dependent_job_name: str) -> List["FakeTrigger"]:  # type: ignore
+        if dependent_job_name:
+            triggers = []
+            for trigger in self.triggers.values():
+                for action in trigger.actions:
+                    if ("JobName" in action) and (
```

Comment:
Would you mind adding a test for this? It looks OK, but it's always good to explicitly verify that passing the argument works as expected

Reply:
@bblommers thanks for pointing that out 👌  I added tests in the next commit.